tuna/rocmlir/rocmlir_worker.py

```diff
@@ -161,7 +185,6 @@ def run_cmd(self):
 
   def get_mlir_v(self) -> str:
     """Interface function to get mlir version info"""
-    #_, mlir_ver, _ = self.exec_docker_cmd("cat /opt/rocm/.info/version")
-    mlir_ver = "mlir_v-not-yet-implemented"
-    self.logger.info('Got mlir version: %s', mlir_ver)
-    return mlir_ver
+    _, mlir_hash, _ = self.exec_docker_cmd("git rev-parse HEAD")
```

Comment:
Finally getting the commit hash of rocMLIR for the version.  Makes me a little nervous to assume what directory we're in, but that happens a lot in this system.

Reply:
Do you have the name of the assumed dir? You could add a check beforehand

Reply:
I'm considering making options to specify tuna and rocMLIR directories.  They won't have fixed names:  some CI machines use /home/jenkins, some use /var/z1_miciadmin.